apps/ledger-live-mobile/src/screens/PTX/Earn/EarnInfoDrawer.tsx

```diff
@@ -19,31 +22,50 @@ export function EarnInfoDrawer() {
     await dispatch(setEarnInfoModal({}));
     await setModalOpened(false);
   }, [dispatch]);
-  const { message, messageTitle } = useSelector(earnInfoModalSelector);
+  const { message, messageTitle, learnMoreLink } = useSelector(earnInfoModalSelector);
 
   useEffect(() => {
     if (!modalOpened && (message || messageTitle)) {
       openModal();
     }
   }, [openModal, message, messageTitle, modalOpened]);
 
+  const onLearMorePress = useCallback(() => {
```

Comment:
```suggestion
  const onLearnMorePress = useCallback(() => {
```
(or `onPressLearnMore`...)